features/ui/select/select.tsx

```diff
@@ -0,0 +1,106 @@
+import styles from "./select.module.scss";
+import { useState } from "react";
+import classNames from "classnames";
+
+type SelectProps = {
+  label: string;
+  options: string[];
+  withIcon?: boolean;
+  hint?: string;
+  errorMessage?: string;
+  isDisabled?: boolean;
+  isError?: boolean;
+};
+
+export function Select({
```

Comment:
Nice job! It's good practice to build a select component yourself but usually you get something wrong when it comes to accessibility or keyboard navigation. So I'd recommend using a headless UI component next time